pycti/utils/opencti_stix2.py

```diff
@@ -2225,9 +2220,6 @@ def export_selected(
             "id": "bundle--" + str(uuid.uuid4()),
             "objects": [],
         }
-        if entity_type == "StixFile":
```

Comment:
If I remember well, this need to be kept

Reply:
I confirm
https://github.com/OpenCTI-Platform/client-python/issues/385

Reply:
It's handle by generate_export method
So, it's ok ✅